docs/getting_started/why_use_GTFS.md

```diff
@@ -1,42 +1,104 @@
+- **Multi-Agency Trips:** GTFS enables agencies to share consistent, reliable data, facilitating seamless travel experiences for riders traveling across multiple transit systems.
+
+## Simplified Data Sharing
+
+GTFS fosters collaboration between transit agencies by enabling easy data exchange, promoting smoother service integration, and information sharing. Here's how GTFS simplifies data sharing:
+
+- **Collaborative Development:** GTFS is collaboratively developed, ensuring the format remains relevant to meet the evolving needs of riders and agencies.
+- **Ease of Integration:** GTFS makes it easy for agencies to get started with a simple data structure using common file formats like .txt and GeoJSON, fostering collaboration and interoperability.
+
+## Wider User Base for Apps & Services
+
+By adopting GTFS, agencies reach a wider audience through the vast pool of app developers utilizing the standardized data format. Here's how GTFS expands the user base for apps and services:
+
+- **Expanded App Reach:** Access to GTFS data allows app developers to reach a wider audience of transit riders, enhancing the usability and functionality of transit apps.
+- **Interoperability:** Open standards facilitate seamless data exchange between transit agencies and app developers, promoting interoperability and collaboration.
+
+## Open Data: More Opportunity and Choices
+
+Unlocking the potential of open data, GTFS offers numerous benefits for both agencies and riders:
+
+- **Wide Accessibility:** GTFS serves as an Open Standard, enabling agencies to easily share transit information through various tools supporting GTFS.
+- **Diverse Applications:** This fosters a broad spectrum of applications and services, ranging from popular trip planners to innovative solutions.
+- **Rider Empowerment:** Riders enjoy the freedom to choose the app that best fits their needs, whether prioritizing accessibility features, simplicity, or cutting-edge innovation.
+
+## Maintained by the Community
+
+GTFS thrives on community collaboration, ensuring its continuous evolution and relevance:
+
+- **Open Collaboration:** GTFS operates as an open data standard, allowing anyone to contribute changes or improvements.
+- **Guided Evolution:** The specification evolution is overseen by an independent nonprofit organization called [MobilityData](https://mobilitydata.org/), and its changes are guided by the principles of ease of use, backwards-compatibility, and pragmatic change. The specification is primarily concerned with passenger information.
+- **Community-driven Development:** The GTFS specification undergoes continuous refinement through input and review from the community, facilitated on platforms like [GitHub](https://github.com/) and [Slack](https://share.mobilitydata.org/slack).
+
+<div class="grid cards" markdown>
```

Comment:
The cards render a bit off when displaying the window fullscreen on a 13-in macbook, do you know if theres any trick to harmonize the width of the cards? the only workaround that I found was to write longer descriptions so that they max out their width, probably not ideal in this case.

<img width="1409" alt="Screenshot 2024-05-28 at 5 01 12 PM" src="https://github.com/MobilityData/elrond.gtfs.org/assets/104692200/2d05176d-c1b2-435d-a6a6-944edf4ce14a">

Reply:
We would have to investigate. Maybe having invisible characters like spacebar would help?

Reply:
@eliasmbd I've tried for another page and it didn't work 😕